assets/pages/digital-subscription-landing/digitalSubscriptionLanding.jsx

```diff
@@ -66,6 +73,18 @@ const content = (
       />
       <ProductBlock countryGroupId={countryGroupId} />
       <AdFreeSection headingSize={2} />
+      <CtaAbTestWrapper>
+        <div />
```

Comment:
our `optimizeTestWrapper` switches between children depending on the AB variant. In this case I want to make this disappear altogether in a variant. if we had fragments I'd use those but since we don't this was a cheap way to get there. I think it's a reasonable compromise given this is AB test and will go away in the next 1-2 sprints